### PR TITLE
Add/remove missing I/O, deleting unused ents, and more behind tags

### PR DESCRIPTION
**Pull request by tmob03**
Resolves [#1912](https://github.com/momentum-mod/game/issues/1912) although the OnTrigger output could be renamed in code to be easier to understand, and then renamed here.

`notlookingfrequency` was inherited from Black Mesa's FGD by accident. `LookTime` and `FieldOfView` inputs were missing from the FGD for some reason, and `OnTrigger` was defined twice, once in base and once here.

### CODE REVIEW

**Comment by tmob03** [comment]
I/O found with `ent_info` in both games. `func_elevator`, `info_elevator_floor`, `math_calculator`, and `point_surroundtest` were not in either games, let me know if they should be behind some tag that aren't compiled into either game, instead.

File: fgd/point/point/point_posecontroller.fgd [hunk 1]
```diff
@@ -35,4 +35,5 @@
 	input SetFModRate(float) : "Set the modulation cycles per second from -10.0f to 10.0."
 	input SetFModAmplitude(float) : "Set the modulation extents from 0.0f to 10.0."
 	input RandomizeFMod(float) : "Randomize the frequency modulation by an extremeness of 0.0 to 1.0."
+	input GetFmod(void) : "Outputs the current modulation settings in console."
```
Comment (braem) [inline fgd/point/point/point_posecontroller.fgd:38]: All this does is print a dev-only message to console - I think it's fine to keep out of the FGD.

File: fgd/point/logic/logic_relay.fgd [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 	// Inputs
 	input Trigger(void) : "Trigger the relay, causing its OnTrigger output to fire if it is enabled."
-	input TriggerWithParameter(string) : "Triggers the relay with a parameter, causing its OnTriggerParameter output to fire if it is enabled."
+	input bTriggerWithParameter(string) : "Triggers the relay with a parameter, causing its OnTriggerParameter output to fire if it is enabled."
```
[the remark this answers is not in the document]
Comment (braem) [reply fgd/point/logic/logic_relay.fgd:17]: Looking at this more, I accidentally renamed it to this during a refactor 🙃

File: fgd/point/point/point_posecontroller.fgd (hunk 1 above)
Comment (tmob03) [reply fgd/point/point/point_posecontroller.fgd:38]: Unless something is clearly broken, or unusable in said game is there much of a reason to keep it out?

File: fgd/point/logic/logic_relay.fgd (hunk 2 above)
[the remark this answers is not in the document]
Comment (tmob03) [reply fgd/point/logic/logic_relay.fgd:17]: > Looking at this more, I accidentally renamed it to this during a refactor 🙃
Lmao, nice

**Comment by tmob03** [comment]
Before merging I believe I found a few more ents not in either game: ``point_hiding_spot, trigger_auto_crouch, func_nav_avoid, func_nav_avoidance_obstacle, func_nav_blocker, and func_nav_prefer` Are these good to remove? Or should some (or all?) be behind another tag?

**Comment by braem** [comment]
Hmm that's strange that these aren't in p2ce. Only one I'd delete is `trigger_auto_crouch`, as p2ce should have the nav ones.

**Review by braem — APPROVED** [review]

**Pull request merged**